components/view-components/VideoComponent.vue

```diff
@@ -36,73 +36,68 @@ const props = defineProps({
 
 <template>
   <div class="video-component">
-    <div class="container">
```

Comment:
Removed the container as each section is already wrapped in a container.
You should never have a container inside another container.
If you need additional columns inside existing columns you just put them inside a row since that counter acts the columns padding and then the nested columns will have it's own padding.
Hope it makes sense otherwise let me know and I will try to elaborate further. ;-)